src/index.js

```diff
@@ -0,0 +1,34 @@
+const BigNumber = require("bignumber.js");
+
+const { getInputOutputFiles, parseCsvFile, writeJson } = require("./utils");
+const GNO_ADDRESS = "0x6810e776880C02933D47DB1b9fc05908e5386b96";
+
+function toPayment(leaderBoardItem) {
+  const { trader, gno_estimation: gnoEstimation } = leaderBoardItem;
```

Comment:
I guess this unpacking is the most import part of row -> transfer Object conversion. My interpretation of this is that `trader` was one of the column headers so we don't need to assign it a new variable name (although I might propose we directly assign it to `receiver` here and `gno_estimation` is the column header, but we reassign it to `gnoEstimation` here simply because JS is generally camelCase. I fee like this would be a bit more intuitive if we directly assigned these unpacked values to what they will become in the transfer object (i.e. `receiver` and `amount` respectively). Just an idea.

Reply:
> Just an idea.

A good idea :)

Actually, Leandro did a similar comment about `gnoEstimation` and I addressed that and did the change to `trader` too